### DISCUSSION TITLE
State of the library

### DISCUSSION THREAD

**Discussion by deeprobin** [locka99/opcua#282] [Q&A] [answered]
Hello,

what is the maintenance status of this repository?
I am asking if this library is the right one for production use.

Is this project just a hobby and are you interested to continue this project for a longer time?

The reason I ask is that I am developing an application for a larger customer, which I have to provide with a SLA.

I've seen it happen many times that projects were shut down because the maintainer didn't have time anymore or wasn't dedicated to the open-source community.

**Comment by locka99** [comment, marked as answer]
I'm doing little pieces of work but not full time at the moment, at some point some pubsub functionality will go in which I've been working on in a branch. Whether it's right for you is something you have to decide for yourself. Contributions are welcome but I'm not I'm not a company or sponsored by one so work happens when it happens at the moment.

**Comment by HerrMuellerluedenscheid** [comment]
Hi @deeprobin, I don't know if its still of importance but we opened an issue (though it probably have been a discussion here...) #414 Feel free to share your experience there. I find the point you raised here quite important.
